elements/dash-tooltip/demo.html

```diff
@@ -0,0 +1,37 @@
+<!doctype html>
+
+<style>
+  /* When the tooltip is hidden, it shouldn't be visible. */
+  dash-tooltip[aria-hidden="true"] {
+    display: none !important;
+  }
+
+  /* The tooltip is by default unstyled. */
+  dash-tooltip {
+    margin-left: 8px;
+    padding: 4px;
+    background-color: #616161;
+    color: #fff;
+    border-radius: 3px;
+    opacity: 0.9;
+
+    /* Optionally, you can animate the tooltip. */
+    -webkit-animation: fadein 1s;
```

Comment:
I’d vote for removing the prefix’d properties. It’s more visual noise than anything else imo.